Replace npm with yarn in exo-clone

martinjaime opened the pull request:
<!-- mention the issue this PR addresses -->
Addresses Originate/exosphere-sdk#170

<!-- a short description of the change in addition to what is already mentioned in the issue above -->

<!-- tag a few reviewers -->

Commit:
Replace npm with yarn in exo-clone

kevgo reviewed (commented):
Same here, please install yarn in case it is missing

Commit:
Update setup and test files

handle if yarn is not installed

Commit:
Merge branch 'master' into mj-yarn-exo-clone

Commit:
Updates script and config

bin/setup -> to be simpler and more compatible
dependency-lint.yml -> workaround bug
